README.md

```diff
@@ -144,6 +145,31 @@ To run the tests on the master node, the pod needs to be scheduled on that node.
 
 The default labels applied to master nodes has changed since Kubernetes 1.11, so if you are using an older version you may need to modify the nodeSelector and tolerations to run the job on the master node.
 
+
+### Running in an AKS cluster
+
+1. Create an AKS cluster(e.g. 1.13.7) with RBAC enabled, otherwise there would be 4 failures
+
+1. Use the [kubectl-enter plugin] (https://github.com/kvaps/kubectl-enter) to shell into a node 
+`
+kubectl-enter {node-name}
+` 
+or ssh to one agent node
+could open nsg 22 port and assign a public ip for one agent node (only for testing purpose)
+
+1. config KUBECONFIG on the agent node:
+```
+sudo chmod a+r /etc/kubernetes/certs/client.key
+export KUBECONFIG=/var/lib/kubelet/kubeconfig
+kubectl get no  # to check whether kubeconfig works
+```
+4. Run CIS benchmark to view results:
+```
+docker run --rm -v `pwd`:/host andyzhangx/kube-bench:latest install
```

Comment:
shouldn't this be `aquasecurity/kube-bench:latest`?

Reply:
Thanks @lizrice .. We had to make some specific changes to the kube-bench code . Should we publish an image with an Azure tag instead ? kube-bench:azure

@andyzhangx

Reply:
Let's see if we can find a way to incorporate the changes into this repo - can you share what needed to change?

Reply:
@andyzhangx Can you please share the diff?

Reply:
the code change is here: https://github.com/andyzhangx/kube-bench/commit/5f5ee060445b762ae93945a8288e7de0c20caf9d
@lizrice